test/models/UserSpec.scala

```diff
@@ -0,0 +1,104 @@
+package models
+
+import models.input._
+import org.h2.jdbc.JdbcBatchUpdateException
+import org.scalatest.{BeforeAndAfterEach, FlatSpec, Matchers}
+import utils.WithInMemoryDB
+
+import java.time.LocalDateTime
+
+class UserSpec extends FlatSpec with Matchers with BeforeAndAfterEach with WithInMemoryDB {
+
+  private val users = Seq(
+    User.create(None, None, "good rule", exported = false),
+    User.create(None, Some("tenant"), "MO", exported = true),
+    User.create(Some(index.id), Some("tenant"), "MO_DE", exported = true),
```

Comment:
not sure I understand how this is working???    maybe not completed yet???

Reply:
should be more reasonable now...